Assets/Scripts/CurrencySystem/CurrencySystem.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine;
+public class CurrencySystem : ScoreSystem
+{
+     [SerializeField] private float convertionMultiplier;
```

Comment:
```suggestion
   [SerializeField] private float convertionMultiplier;
```
Zorg voor goede indentation